google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -834,6 +834,15 @@ public static BlobListOption currentDirectory() {
     }
 
     /**
+     * Returns an option for bucket's billing user project. This option is only used by the buckets with
```

Comment:
This option is required by buckets with `requester_pays` flag enabled.

Reply:
Done.